src/config/config.c

```diff
@@ -559,57 +570,72 @@ static void initConfig(struct config *conf)
+	conf->dns.localise.a = cJSON_CreateStringReference("<true|false>");
 	conf->dns.localise.t = CONF_BOOL;
 	conf->dns.localise.f = FLAG_RESTART_FTL;
 	conf->dns.localise.d.b = true;
 	conf->dns.localise.c = validate_stub; // Only type-based checking
 
+	conf->dns.revServers.k = "dns.revServers";
+	conf->dns.revServers.h = "Reverse server (formerly called \"conditional forwarding\")\n\n If not configured as your DHCP server, Pi-hole typically won't be able to determine the names of devices on your local network. As a result, tables such as Top Clients will only show IP addresses.\n\n One solution for this is to configure Pi-hole to forward these requests to your DHCP server (most likely your router), but only for devices on your home network. To configure this we will need to know the IP address of your DHCP server and which addresses belong to your local network.\n\n Example: [ \"true,192.168.0.0/24,192.168.0.1,fritz.box\" ]";
+	conf->dns.revServers.a = cJSON_CreateStringReference("Array of reverse servers each one in the following form:\n\n \"<enabled>,<ip-address>[/<prefix-len>],<server>[#<port>][,<domain>]\"\n\nExplanation of individual components:\n\n - \"<enabled>\": either \"true\" or \"false\"\n\n - \"<ip-address>[/<prefix-len>]\": Address range for the reverse server feature in CIDR notation. If the prefix length is omitted, either 32 (IPv4) or 128 (IPv6) are substituted (exact address match). This is almost certainly not what you want here.\n Example: \"192.168.0.0/24\" for the range 192.168.0.1 - 192.168.0.255\n\n - \"<server>[#<port>]\": Target server to be used for the reverse server feature\n Example: \"192.168.0.1#53\"\n\n - \"<domain>\": Domain used for the reverse server feature (e.g., \"fritz.box\")\n Example: \"fritz.box\"");
+	conf->dns.revServers.t = CONF_JSON_STRING_ARRAY;
+	conf->dns.revServers.d.json = cJSON_CreateArray();
+	conf->dns.revServers.c = validate_dns_revServers;
+	conf->dns.revServers.f = FLAG_RESTART_FTL;
+
 	// sub-struct dns.cache
 	conf->dns.cache.size.k = "dns.cache.size";
-	conf->dns.cache.size.h = "Cache size of the DNS server. Note that expiring cache entries naturally make room for new insertions over time. Setting this number too high will have an adverse effect as not only more space is needed, but also lookup speed gets degraded in the 10,000+ range. dnsmasq may issue a warning when you go beyond 10,000+ cache entries.";
+	conf->dns.cache.size.h = "Cache size of the DNS server. Note that expiring cache entries naturally make room for new insertions over time.\n\nSetting this number too high will have an adverse effect as not only more space is needed, but also lookup speed gets degraded in the 10,000+ range. dnsmasq may issue a warning when you go beyond 10,000+ cache entries.";
+	conf->dns.cache.size.a = cJSON_CreateStringReference("A positive integer value");
```

Comment:
Similarly here, we could add these `a = cJSON_CreateStringReference("A positive integer value");` at the end when the type matches and the existing `.a == NULL`. It'd trivially ensure it cannot be forgotten in the future.

Reply:
We spoke about this one on MM - leaving it for the time being